Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+
+.. _vendor_team: https://github.com/orgs/cilium/teams/vendor
+.. _go_dot_mod: https://github.com/cilium/cilium/blob/main/go.mod
+.. _go_dot_sum: https://github.com/cilium/cilium/blob/main/go.sum
+.. _vendor_slash: https://github.com/cilium/cilium/blob/main/vendor
+.. _cilium_slash_cilium_reviews: https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_hubble_reviews: https://github.com/cilium/hubble/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_cilium_cli_reviews: https://github.com/cilium/cilium-cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_certgen_reviews: https://github.com/cilium/certgen/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _ladder: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md
+
+Reviewing Pull Requests
+=======================
+
+This section describes some of the process and expectations for reviewing PRs
+on behalf of @cilium/vendor. Note that :ref:`the generic PR review
+process for Committers <review_process>` still applies, even though it is not
+specific to dependencies.
+
+Existing Dependencies
+---------------------
+
+Updates to existing dependencies most commonly occur through PRs opened by
+`Renovate <renovate_>`_, which is a 3rd party service used throughout the
+Cilium organization. Renovate continually checks repositories for out-of-date
+dependencies and opens new PRs to update any it finds.
+
+.. note::
+   Renovate is configured via a file named ``renovate.json5`` found within the
+   ``.github/`` folder of each repository it is enabled on. Cilium's Renovate
+   configuration is long and complex, having been developed to enable
+   dependency updates to Docker images and GitHub Actions. Only a few pieces
+   of the configuration are relevant to the @cilium/vendor team.
+
+.. _renovate: https://docs.renovatebot.com
+
+When reviewing PRs that update an existing dependency, members of the
+@cilium/vendor team are required to ensure that the update does not include
+any breaking changes or licensing issues. These checks are facilitated via
+GitHub Action CI workflows, which are triggered by commenting ``/test`` within
+a PR. See :ref:`CI  / GitHub Actions <ci_gha>` for more information on their
+use.
+
+New Dependencies
+----------------
+
+When a new dependency is added as part of a PR, the @cilium/vendor team will
+be assigned to ensure the new dependency meets the following criteria:
+
+1. The new dependency must add functionality that is not already provided within
+   an existing dependency, an internal package to the project, or Go's
+   standard library.
+2. The functionality provided by the new dependency must be non-trivial to
+   re-implement manually.
+3. The new dependency must be actively maintained, having new commits and/or
```

Comment:
I expanded a bit on the fourth bullet point mentioning some criteria to check for. Listing out exact criteria is a bit tough because not all projects implement the same kind of best practices; there's a lot of different factors so it's a bit of a gray area. I'd rather air on the side of being broad rather than too specific, and let discussions for whether or not a dependency is appropriate happen in a PR. WDYT?

Reply:
I think it make sense, maybe if we end up having that monthly meeting it would even make much more sense, thanks.

Reply:
I think these guidelines should be good enough as a basis. If we need to challenge them while evaluating how to introduce a necessary dependency that doesn't fit well into the criteria, we can have the discussion then about how to approach it.